test/evergreen.yml

```diff
@@ -2256,7 +2256,7 @@ tasks:
           script: |
             set -o errexit
             set -o verbose
-            ${python_binary|python3} syscall.py --verbose --preserve
+            WT_BUILDDIR=$(pwd)/../../cmake_build LD_LIBRARY_PATH=$WT_BUILDDIR ${python_binary|python3} syscall.py --verbose --preserve
```

Comment:
Can we use  `${test_env_vars|}` instead of `WT_BUILDDIR=$(pwd)/../../cmake_build LD_LIBRARY_PATH=$WT_BUILDDIR` here?

Reply:
Unfortunately this won't quite work. Sometimes the `test_env_vars` variable is setup with environment variables that will break the test. On specific example is using`LD_PRELOAD=/usr/local/lib/libeatmydata.so`. This is unfortunately impacts/breaks the `syscall-linux` runtime, failing the test. 
For this specific case, its better to be explicit about the variables the test requires, keeping the runtime environment as clean as possible for `syscall.py`